src/index.scss

```diff
@@ -1,6 +1,48 @@
+$base-font-color: #616161;
+$primary-font-color: rgba(0, 0, 0, 0.5);
+$primary-spacing: 5px;
+
 body {
   margin: 0;
   padding: 0;
-  font-family: sans-serif; }
+  font-family: Merriweather;
+  font-weight: 300;
+  font-style: normal;
+  font-size: 15px;
```

Comment:
Use variables for font sizes and derive some font-based properties (like `line-height`) from this var.